src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -44,6 +44,8 @@
     private static final String STARTING_STATE = DecisionsStates.DECISION_DRAFTED.getStateName();
 
     private static final String PENDING_STATE = DecisionsStates.DECISION_ISSUE_PENDING.getStateName();
+    private static final String ONLINE_HEARING_NOT_FOUND = "Online hearing not found";
```

Comment:
Let's move ONLINE_HEARING_NOT_FOUND somewhere common as it being used in so many places